content/v2.0/reference/flux/stdlib/built-in/transformations/aggregates/count.md

```diff
@@ -21,6 +21,18 @@ _**Output data type:** Integer_
 count(column: "_value")
 ```
 
+{{% note %}}
+#### Count empty tables
+`count()` returns `0` for empty tables.
+To keep empty tables in your data, set the following parameters for the following functions:
+
+| Function                                                                                              | Parameter           |
+|:--------                                                                                              |:---------           |
+| [filter()](/v2.0/reference/flux/stdlib/built-in/transformations/filter/)                              | `onEmpty: "keep"`   |
```

Comment:
Can you add a warning here that keeping empty data on your first filter can have pretty severe performance costs? If the person wants to keep tables from a filter, they should do their normal filters first and then do whatever filter where they want to keep the empty tables as a second call to filter.